flood_adapt/object_model/direct_impacts.py

```diff
@@ -126,6 +126,8 @@ def run_models(self):
         end_time = time.time()
         print(f"Running FIAT took {str(round(end_time - start_time, 2))} seconds")
 
+        self.postprocess_fiat()
```

Comment:
Why was this added here? I think the whole idea was to seperate preproccess, run, and postprocess to different methods. 

The scenario.run() method takes care of the workflow.
```suggestion
```

Reply:
You're right, some stuff went wrong with the rebase apparently